clients/src/test/java/org/apache/kafka/common/network/SslTransportLayerTest.java

```diff
@@ -551,6 +551,35 @@ public void testInvalidKeyPassword() throws Exception {
         server.verifyAuthenticationMetrics(0, 1);
     }
 
+    /**
+     * Tests that connection sucess with the default TLS version.
+     */
+    @Test
+    public void testTLSDefaults() throws Exception {
+        final LogContext logContext = new LogContext();
+
+        sslServerConfigs = serverCertStores.getTrustingConfig(clientCertStores);
+        sslClientConfigs = clientCertStores.getTrustingConfig(serverCertStores);
+
+        assertEquals(SslConfigs.DEFAULT_SSL_PROTOCOL, sslServerConfigs.get(SslConfigs.SSL_PROTOCOL_CONFIG));
+        assertEquals(SslConfigs.DEFAULT_SSL_PROTOCOL, sslClientConfigs.get(SslConfigs.SSL_PROTOCOL_CONFIG));
```

Comment:
Not sure if these configs actually come from the default configs. I think these may be explicitly configured in the tests.

Reply:
This comes from `TestSslUtils#DEFAULT_TLS_PROTOCOL_FOR_TESTS` - chaned it to

```
    public static final String DEFAULT_TLS_PROTOCOL_FOR_TESTS = SslConfigs.DEFAULT_SSL_PROTOCOL;
```